substrafl/remote/register/register.py

```diff
@@ -57,6 +55,9 @@
 {local_dependencies}
 
 ENTRYPOINT ["python{python_version}", "function.py", "--function-name", "{method_name}"]
+
+COPY . .
```

Comment:
Aren't we copying twice the dependency dir and the wheel dir by doing that way ?

Reply:
yes, the local dependency dir and the dir containing the wheels
substrafl and subtra wheels are less than 100kB, so probably not worth an optimisation (compared to the 5 GB of torchvision for example)
I don't know what the local dependency dir may contain 🤷  Do we have an example somewhere?

Reply:
It can contains a whole lib. For instance, on the Flamby example, the Flamby repo is a local dependency and might be several Gb.. https://github.com/owkin/FLamby/blob/main/integration/Substra/flamby-integration-into-substra.md#:~:text=local_dependencies%3D%5Bpathlib.Path.cwd().parent.parent%5D%2C%20%23%20Flamby%20dependency

Reply:
This is the old asana card : https://app.asana.com/0/1190766234519625/1202168155731350/f
No usefull info though... How urgent is the merge of this PR ?
I can approve it and we'll create a new card to investigate this "explicit COPY in dockerfile" in a less urgent context ?

Reply:
Well I guess there are two options here. 
One is to provide an exhaustive list of files we want to copy 🤔 (`Dockerfile`, `function.py`, and a part of `substrafl_internals`). 
The other is to change the file structure so that everything is not bundled under `substrafl_internals`. 
I'm not sure which one is the best.

Reply:
Agree :) I'll create the card